From a87486e18e900271a7bd44453ce7c33b33bb78d3 Mon Sep 17 00:00:00 2001
From: Jonathan Lebon <jonathan@jlebon.com>
Date: Thu, 25 Oct 2018 14:52:12 -0400
Subject: [PATCH] rust: Drop crates-io patch and use 0.4.0

The latest release of the crate includes the features and fixes we need.

Closes: #1639
Approved by: cgwalters
---
 rust/Cargo.toml | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/rust/Cargo.toml b/rust/Cargo.toml
index 202d2287..62aa51a4 100644
--- a/rust/Cargo.toml
+++ b/rust/Cargo.toml
@@ -16,11 +16,7 @@ tempfile = "3.0.3"
 openat = "0.1.15"
 curl = "0.4.14"
 c_utf8 = "0.1.0"
-systemd = "0.3.0"
-
-# Until https://github.com/jmesmon/rust-systemd/pull/54 gets merged
-[patch.crates-io]
-systemd =  { git = "https://github.com/jlebon/rust-systemd", branch = "pr/add-monotonic" }
+systemd = "0.4.0"
 
 [lib]
 name = "rpmostree_rust"
-- 
2.17.1

